src/dftbp/dftb/energytypes.F90

```diff
@@ -121,6 +129,9 @@ module dftbp_dftb_energytypes
     !> Atom resolved spin
     real(dp), allocatable :: atomSpin(:)
 
+    !> Atom resolved Multipole
+    real(dp), allocatable :: atomDftbMultiExpan(:)
```

Comment:
Will this be for the total multipolar energy (i.e. everything except monopole-monopole)? The comment probably should reflect this if so.